content/guides/building/get-source-git.md

```diff
@@ -116,16 +127,16 @@ Haiku's source code needs to reside on a case sensitive file system.
 In short, such a file system recognizes "ThisIsAFile.txt" and "THISISAFILE.txt" as two different files. Some file systems that are (or could be) case in-sensitive include, FAT32, NTFS, and HFS+. macOS's HFS+ is case in-sensitive by default. For more information regarding how to create a case-sensitive HFS+ volume, see <a href="/documents/dev/how_build_haiku_mac_os_x#part_diskimage">this article</a>.
 
 <a name="proxy_access"></a>
-<h4>Getting the source code through an HTTP proxy</h4>
+<h4>Obtaining the source code using an HTTP proxy</h4>
 
 <div class="alert alert-warning">
 Haiku's main Git repository does not allow HTTP access, which is a problem if you are accessing the Internet through a proxy server that only permits HTTP (port 80) traffic.
 </div>
 
-Instead, use one of our mirror repositories at GitHub or Gitorious for anonymous HTTP access, they are both kept in sync with the main repository. First, set Git to connect through your proxy server:
+Use one of our mirror repositories <a href="https://github.com/haiku/haiku">at GitHub</a> for anonymous HTTP access instead. They are both kept in sync with the main repository. First, set Git to connect through your proxy server:
```

Comment:
```suggestion
Use our mirror repositories <a href="https://github.com/haiku/haiku">at GitHub</a> for anonymous HTTP access instead. They are kept in sync with the main repository. First, set Git to connect through your proxy server:
```

Reply:
What other mirror repositories are there?

Reply:
I'm not aware of other mirrors, that's why I just removed the "both" here which referred to github and gitorious before.

Reply:
> Use one of our mirror repositories

... is still there, I'll commit the change and fix this later.